### DISCUSSION TITLE
Status of MoltenVK CTS Compliance (dEQP)?

### DISCUSSION THREAD

**Discussion by jdd510** [KhronosGroup/MoltenVK#1469] [Q&A]
Hi, I work on the Android Emulator team. We are trying to pass CTS on MacOS right now, but it seems some of the dEQP-VK tests are failing. I see there is some work going on with CTS compliance in MoltenVK, and was wondering what the status and timeline of this work is. Thank you.

**Comment by billhollings** [comment]
Issue #1285 tracks MoltenVK's CTS status.
